src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -259,5 +259,61 @@ public UserManagementService(WorkOSClient client)
             };
             return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
         }
+
+        /// <summary>
+        /// Update the user's information.
+        /// </summary>
+        /// <param name="options"> Parameters used to update a user.</param>
+        /// <param name="cancellationToken">An optional token to cancel the request.</param>
+        /// <returns> The corresponding User object.</returns>
+        public async Task<User> UpdateUser(
+            UpdateUserOptions options,
+            CancellationToken cancellationToken = default)
+        {
+            var request = new WorkOSRequest
+            {
+                Options = options,
+                Method = HttpMethod.Put,
+                Path = $"/users/{options.Id}",
```

Comment:
@maxdeviant  Getting your opinion on new way of passing `id`, think it makes more sense to pass id through the options interface than having 2 seperate params in `id` and `options`, if this doesn't make sense I can revert back to previous method. If it makes sense I'll pull another PR to fix across the SDK

Reply:
One difference with this approach is that now `id` will get passed on `Options` as well in the serialized form, so we'll have to make sure to explicitly mark it to be excluded from serialization.

Reply:
Right, think we should be able to just add [JsonIgnore] to those fields to ignore serialization then.